pkg/status/dist/templates/logsagent.tmpl

```diff
@@ -4,6 +4,11 @@ NOTE: Changes made to this template should be reflected on the following templat
 */}}==========
 Logs Agent
 ==========
+{{- if .warnings }}
```

Comment:
I think we should add this block after `line 9` because can't raise warnings if the agent is not running